doc/_extensions/odoo_ext/translator.py

```diff
@@ -515,6 +515,20 @@ def visit_reference(self, node):
         self.body.append(self.starttag(node, 'a', **attrs))
     def depart_reference(self, node):
         self.body.append(u'</a>')
+    def visit_download_reference(self, node):
+        # type: (nodes.Node) -> None
+        if node.hasattr('filename'):
+            self.body.append(
+                '<a class="reference download internal" href="%s" download="">' %
+                posixpath.join(self.builder.dlpath, node['filename']))
```

Comment:
This should use the `starttag` method to ensure everything is properly sanitised & normalised.